projects/storefrontlib/src/cms-components/configurator/commons/configure-product/configure-product.component.html

```diff
@@ -3,7 +3,7 @@
     [routerLink]="
       {
         cxRoute: 'configure' + configuratorType,
-        params: { rootProduct: productCode }
+        params: { ownerType: 'product', entityKey: productCode }
```

Comment:
Can we use the Enumeration instead of hard coded 'product'?

Reply:
Done